modules/git/command.go

```diff
@@ -278,3 +278,19 @@ func (c *Command) RunTimeout(timeout time.Duration) (string, error) {
 func (c *Command) Run() (string, error) {
 	return c.RunTimeout(-1)
 }
+
+// HelperForTestsToAllowLFSFilters should only be used for tests
+func HelperForTestsToAllowLFSFilters() []string {
```

Comment:
```go
func ThatsOneBigVeryDescriptiveNameForASimpleFunctionThatActAsAHelperForTests()
```

Reply:
well I could not add it to command_test.go ... and it should not used outside of tests - that should make it clear :D

Reply:
I think we should just call this `AllowLFSFiltersArgs()` 

By all means stick in the comment that it is intended only to be used in tests but this name is horrendous.

Reply:
done